### PR TITLE
Fix UI warnings

### PR DESCRIPTION
**Pull request by zkhin**

### CODE REVIEW

File: src/scenes/Products/Detail/Progress/StepProgress.tsx [hunk 1]
```diff
@@ -36,7 +36,7 @@ export const StepProgress = ({
       </Grid>
 
       <Grid item xs>
-        <Card variant="outlined">
+        <Card variant="outlined" elevation={0}>
```
Comment (zkhin) [inline src/scenes/Products/Detail/Progress/StepProgress.tsx:39]:
```
Warning: Failed prop type: MUI: Combining `elevation={8}` with `variant="outlined"` has no effect. Either use `elevation={0}` or use a different `variant`
```

File: src/components/ProjectChangeRequestListItem/ProjectChangeRequestListItem.tsx [hunk 2]
```diff
@@ -50,7 +50,7 @@ export const ProjectChangeRequestListItem = ({
     <Card className={className}>
       <CardContent>
         <Typography
-          variant="body1"
+          variant="subtitle1"
```
Comment (zkhin) [inline src/components/ProjectChangeRequestListItem/ProjectChangeRequestListItem.tsx:53]:
```
 <p> cannot be under another <p>
```

Comment (CarsonF) [reply src/components/ProjectChangeRequestListItem/ProjectChangeRequestListItem.tsx:53]: The component prop can be used to change the tag without changing the variant prop's styling.

File: src/scenes/Products/Detail/ProductDetailHeader.tsx [hunk 3]
```diff
@@ -41,31 +41,33 @@ export const ProductDetailHeader = ({ product }: { product?: Product }) => {
...
-            <Fab
-              color="primary"
-              // @ts-expect-error it works. These generics are hard to express.
-              component={Link}
-              to="edit"
-            >
-              <Edit />
-            </Fab>
-          </Tooltip>
+      <Grid item>
```
Comment (zkhin) [inline src/scenes/Products/Detail/ProductDetailHeader.tsx:44]: I think `<Grid item container ...>` should be wrapped with `<Gird item>` if the item container is in the `<Grid container>`

File: src/components/ProjectChangeRequestListItem/ProjectChangeRequestListItem.tsx (hunk 2 above)
Comment (CarsonF) [reply src/components/ProjectChangeRequestListItem/ProjectChangeRequestListItem.tsx:53]: Maybe the ones below get `component="span"` instead.

File: src/scenes/Engagement/LanguageEngagement/Header/LanguageEngagementHeader.tsx [hunk 4]
```diff
@@ -70,70 +70,76 @@ export const LanguageEngagementHeader = ({
...
-              : {})}
-          >
-            {langName ?? (
-              <Redacted
-                info={`You do not have permission to view this engagement's ${
-                  language ? 'name' : 'language'
-                }`}
-                width="100%"
-              />
+      <Grid item>
```
Comment (CarsonF) [inline src/scenes/Engagement/LanguageEngagement/Header/LanguageEngagementHeader.tsx:73]: Column divider isn't centered.
Screen Shot 2022-08-01 at 10 41 59 AM
Screen Shot 2022-08-01 at 10 43 27 AM

**Review by CarsonF — CHANGES_REQUESTED** [review]

**Pull request merged**